draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
               <list style="symbols">
 
-                <t>An "a=msid" line, as specified in
-                <xref target="I-D.ietf-mmusic-msid"></xref>, Section
-                2.</t>
+                <t>For each MediaStream that was associated with the
+                transceiver when it was created via addTrack or addTransceiver,
+                an "a=msid" line, as specified in
+                <xref target="I-D.ietf-mmusic-msid"></xref>, Section 2. If a
+                MediaStreamTrack is attached to the transceiver's RtpSender,
```

Comment:
Sorry to be nitpicking, but I'm not sure that this is quite right.

Suppose you do addTransceiver() and hence there's no MS or MST associated with the transceiver
Now you do replaceTrack() so there's a track. Don't you still need to generate an MSID line? As Justin points out, if no MS is associated, you need a '-' MSID.